### ISSUE TITLE
ImportError: cannot import name 'BPlistReader' from 'bplist' 

### ISSUE THREAD

**Issue by form-follows-function** [farcaller/bplist-python#8]
```
>>> from bplist import BPlistReader
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'BPlistReader' from 'bplist' (/usr/local/lib/python3.8/site-packages/bplist/__init__.py)
```

Occured on macOS 10.15.7. Installed python 3.8 via homebrew.
A quick look into the  `__init__.py` didn't give me any hint why this occurs.

However this works:
```
from bplist import *

print(dir(bplist))
```

**Comment by unformatt** [comment]
Should be `BPListReader` rather than `BPlistReader`

**Comment by farcaller** [comment]
Fixed in #9
